### PR TITLE
handling of directory selector's cancellation

### PR DESCRIPTION
**Pull request by Trigary**

### CODE REVIEW

File: src/main/java/com/gmail/stefvanschiedev/bfide/application/menu/file/OpenMenuItem.java [hunk 1]
```diff
@@ -30,7 +33,12 @@ public OpenMenuItem(StartStage stage) {
                 return;
             }
 
-            stage.getOpenProjects().add(new Project(directory.getName(), directory));
+            Project project = Project.create(directory, directory.getName());
```
Comment (stefvanschie) [inline src/main/java/com/gmail/stefvanschiedev/bfide/application/menu/file/OpenMenuItem.java]: Project.create creates a project like it doesn't exist yet. This means that a project folder and a .bfide folder will be created. However when opening a project those folder already exists.

**Review by stefvanschie — CHANGES_REQUESTED** [review]
Small change in OpenMenuItem

File: src/main/java/com/gmail/stefvanschiedev/bfide/application/menu/file/OpenMenuItem.java (hunk 1 above)
Comment (Trigary) [reply src/main/java/com/gmail/stefvanschiedev/bfide/application/menu/file/OpenMenuItem.java]: This has been addressed in the next commit (not exactly like that, but that's not due to stubbornness, I promise).

**Review by stefvanschie — APPROVED** [review]

**Comment by stefvanschie** [comment]
Thanks, merged.

**Pull request merged**